src/fdk_organization_bff/service/org_catalog_service.py

```diff
@@ -158,6 +176,7 @@ async def get_organization_catalog(
                 org_datasets=responses[2],
                 assessment_data=responses[3],
             ),
+            dataservices=map_org_dataservices(org_dataservices=responses[4]),
```

Comment:
Det er også mulig å bruke destructuring på responsen fra gather funksjonen for å slippe bruke tall indekser https://riptutorial.com/python/example/14981/destructuring-assignment

Reply:
fancy! 3a5303e